File: week2/fizzbuzzkata.py
# Defining function
def FizzBuzz(num):
    if num == 1:
        return "1"
    elif num == 2:
        return "2"
    elif num % 3 == 0 and num % 5 == 0:
        return "FizzBuzz"
    elif num % 3 == 0:
        return "Fizz"
    elif num % 5 == 0:
        return "Buzz"

# # Displaying results when ran..
# print(FizzBuzz(1))
# print(FizzBuzz(2))
# print(FizzBuzz(3))
# print(FizzBuzz(5))
# print(FizzBuzz(6))
# print(FizzBuzz(10))
# print(FizzBuzz(15))

File: week6/fizzbuzzkata.js
module.exports = (arg) => {
    if (arg === 1) {
        return 1;
    } else if (arg === 2) {
        return 2;
    } else if (arg % 3 === 0 && arg % 5 === 0) {
        return "FizzBuzz";
    } else if (arg % 3 === 0) {
        return "Fizz";
    } else if (arg % 5 === 0) {
        return "Buzz";
    } 
}
